static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -37,3 +37,68 @@ Once initialized in a project, DVC populates its installation directory
 - `.dvc/updater.lock` - a lock file for `.dvc/updater`.
 
 - `.dvc/lock` - a lock file for the whole dvc project.
+
+## Structure of cache directory
+
+The structure of directory depends if the data is stored in a file or is in a
+directory.
+
+In case of a file, it is converted to a MD5 checksum which is a 32 characters
```

Comment:
`it is converted` - it's exact. We calculate md5 and use it as a file name. The file content is preserved.

it's not always MD5, let's do something like `to a checksum (usually MD5 which is ...)`